Rubberduck.UnitTesting/UnitTesting/TestEngine.cs

```diff
@@ -344,7 +454,7 @@ private void RunWhileSuspendedOnUiThread(IEnumerable<TestMethod> tests)
                         {
                             // FIXME somehow notify the user of this mess
                             Logger.Error(ex,
-                                "Unexpected COM exception while cleaning up tests for module {0}. Aborting any further unit tests",
+                                "Unexpected COM expection while cleaning up tests for module {0}. Aborting any further unit tests",
```

Comment:
lol it was correct before?

Reply:
Oops, I "fixed" the wrong one. Fixed in 1b795d71e1c02d9f2bfbd843275b173554506103.